booking/src/main/java/org/dcsa/conformance/standards/booking/BookingScenarioListBuilder.java

```diff
@@ -109,16 +128,15 @@ private BookingScenarioListBuilder thenAllPathsFrom(
       case RECEIVED -> then(
           shipper_GetBooking(bookingState)
               .thenEither(
-                  uc2_carrier_requestUpdateToBookingRequest().thenAllPathsFrom(PENDING_UPDATE),
-                  uc3_shipper_submitUpdatedBookingRequest()
-                      .thenAllPathsFrom(UPDATE_RECEIVED),
-                  uc4_carrier_rejectBookingRequest().thenAllPathsFrom(REJECTED),
-                  uc5_carrier_confirmBookingRequest().thenAllPathsFrom(CONFIRMED),
-                  uc11_shipper_cancelBooking().thenAllPathsFrom(CANCELLED)));
+                auc_shipper_sendInvalidBookingAction(CANCEL_BOOKING_AMENDMENT).then(shipper_GetBooking(bookingState)),
+                uc2_carrier_requestUpdateToBookingRequest().thenAllPathsFrom(PENDING_UPDATE),
+                uc3_shipper_submitUpdatedBookingRequest()
+                    .thenAllPathsFrom(UPDATE_RECEIVED),
+                uc4_carrier_rejectBookingRequest().thenAllPathsFrom(REJECTED),
+                uc5_carrier_confirmBookingRequest().thenAllPathsFrom(CONFIRMED),
+                uc11_shipper_cancelBooking().thenAllPathsFrom(CANCELLED)));
       case START -> thenEither(
-        uc1_shipper_SubmitBookingRequest(ScenarioType.REGULAR).thenAllPathsFrom(RECEIVED),
-        uc1_shipper_SubmitBookingRequest(ScenarioType.REEFER).thenHappyPathFrom(RECEIVED),
-        uc1_shipper_SubmitBookingRequest(ScenarioType.DG).thenHappyPathFrom(RECEIVED));
+        uc1_shipper_SubmitBookingRequest().thenAllPathsFrom(RECEIVED));
```

Comment:
I do not understand what made you remove the other scenario types?

Reply:
Good catch.. I was testing on local machine with only regular scenario hence I had to delete it.. will roll it back.

Reply:
I could recall now, those functions were redundant and not being used any more. Hence removed them.